packages/common-utils/src/types.ts

```diff
@@ -586,6 +582,7 @@ const MetricSourceAugmentation = {
 
   // Metric tables - at least one should be provided
   metricTables: MetricTableSchema,
+  timestampValueExpression: RequiredTimestampColumnSchema,
```

Comment:
I don't recall metric source requires the `timestampValueExpression`. can you explain the change here?

Reply:
Yeah! `timestampValueExpression` was previously defined in `SourceBaseSchema`, which extended by `MetricSourceAugmentation` (and the other 3 types of sources) in `SourceSchema` (and therefore, TSource and TSourceUnion). Since I removed `timestampValueExpression` from `SourceBaseSchema` (because it should no longer be inherited by the Sessions schema), I added it back to the metrics, sources, and traces schemas.

The intent of the change is to maintain the previous schema and types for metrics, logs, and traces.

Reply:
Got it. That makes sense